src/keys/impls.rs

```diff
@@ -22,6 +22,15 @@ impl<G: Group> PublicKey<G> {
         ExtendedCiphertext::new(element, self, rng).inner
     }
 
+    /// Encrypts a group element
+    pub fn encrypt_element<R: CryptoRng + RngCore>(
+        &self, value:
+         G::Element,
```

Comment:
Nit: formatting is incorrect here; `value: G::Element` should be on a single line. Run `cargo fmt --all` locally to have the code formatted automatically.